webapp/pages/post/_id/_slug/index.vue

```diff
@@ -122,7 +132,7 @@ export default {
     UserTeaser,
     HcShoutButton,
     ContentMenu,
-    HcCommentForm,
+    CommentForm,
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-29T16:09:44Z" title="Wednesday, January 29th 2020, 5:09:44 pm +01:00">Jan 29, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -122,7 +132,7 @@ export default {
     UserTeaser,
     HcShoutButton,
     ContentMenu,
-    HcCommentForm,
+    CommentForm,
```


:+1: I like this!

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-29T20:08:37Z" title="Wednesday, January 29th 2020, 9:08:37 pm +01:00">Jan 29, 2020</time>_

---


yeah, I opened a ticket to rename all these in bulk, but I've been sneaking some of these in cause I can't help myself